snippets/logging-fluentd-dep-snip.adoc

```diff
@@ -0,0 +1,12 @@
+// Text snippet included in the following assemblies:
+//
+//
+// Text snippet included in the following modules:
+//
+//
+:_content-type: SNIPPET
+
+[NOTE]
+====
+In Logging 5.6, Fluentd is deprecated and is planned to be removed in a future release. Red Hat will provide bug fixes and support for this feature during the current release lifecycle, but this feature will no longer receive enhancements and will be removed. As an alternative to Fluentd, you can use Vector instead.
```

Comment:
This kinda sounds like we won't be supporting fluentd in Log 5.7 which I don't think is correct.

Reply:
This is the boilerplate deprecation notice, a 'future release' doesn't automatically = 5.7.